scripts/postinstall.js

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env node
+
+const { copySync } = require('fs-extra');
+
+const files = [
+  copySync('node_modules/testpilot-ga/dist/index.js', 'src/vendor/testpilot-ga.js'),
+  copySync('node_modules/quill/dist/quill.min.js', 'src/sidebar/vendor/quill.js'),
+  copySync('node_modules/quill/dist/quill.snow.css', 'src/sidebar/vendor/quill.snow.css'),
+  copySync('node_modules/quill/LICENSE', 'src/sidebar/vendor/quill.LICENSE')
```

Comment:
`fs.copySync()` returns a promise, which allows us to check everything is kosher using `Promise.all()` below, and fail hard if we had an error.